pkgs/development/libraries/science/math/mkl/default.nix

```diff
@@ -55,15 +55,19 @@ stdenvNoCC.mkDerivation rec {
   dontStrip = true;
   dontPatchELF = true;
 
+  # FIXME: On Darwin, we need fixDarwinDylibNames which does not seem
```

Comment:
Ofcorse, all this did was limit the build time dependencies for the path.

Reply:
However the reason it was used isn't true anymore for darwin, so best not to do it. Making this a fixed output doesn't really have any benefits.

Reply:
OK, I'm removing the fixed output them.